app/assets/javascripts/admin/addon/components/schema-theme-setting/editor.gjs

```diff
@@ -38,24 +44,30 @@ export default class AdminThemeSettingSchema extends Component {
     }
 
     const tree = new Tree();
-    const idProperty = schema.identifier;
-    const childObjectsProperties = this.findChildObjectsProperties(
-      schema.properties
-    );
 
-    data.forEach((obj, index) => {
-      const node = new Node({ text: obj[idProperty], index });
+    data.forEach((object, index) => {
+      const node = new Node({
+        index,
+        schema,
+        object,
+        text: object[schema.identifier],
```

Comment:
I think we should make the identifier optional and fallback to the schema name

Reply:
Yeah I agree but I'll get to this in a later PR.